frontend/src/app/login/login.ctrl.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Controller for login form
+ */
+angular.module('flocs.user',[])
+.controller('loginCtrl',['$scope','$location', '$log','userDao',
+	function($scope,$location, $log,userDao){
+        userDao.loggedIn().then(function(response){
+            $scope.user = response.data.username;
+        });
+		function send(){
+			var username = $scope['username'];
+            $log.log($scope['username']);
```

Comment:
can you please indent code everywhere the same, ideally each level by two spaces